app/views/jst/courses/roster/rosterUser.handlebars

```diff
@@ -2,7 +2,11 @@
   {{>avatar}}
 </td>
 <td>
-  <a href="{{html_url}}" class="roster_user_name">{{name}}</a>
+  {{#if canViewSisId}}
+  <a href="{{html_url}}" class="roster_user_name">{{name}}</a> ({{short_name}})
```

Comment:
If `roster_user_name` was left nil with `name` and `short_name` were the same it would probably make the logic a bit easier also.